lib/users.rb

```diff
@@ -0,0 +1,19 @@
+require 'pg'
+
+class User
+
+  def initialize(id:, name:, email:, username:, password:)
+    @id = id
+    @name = name
+    @email = email
+    @username = username
+    @password = password
+end
```

Comment:
I think some of your indentations are slightly off with the 'ends' though that's not a big deal.